File: ConcentraVHM/ClientApp/src/Models/Cliente.ts
export class Cliente {
  nombre: string;
  apellido: string;
  cedula: string;
  fechaNacimiento: string;
  tipoPersona?: string;

  constructor(nombre: string, apellido: string, cedula: string, fechaNacimiento: string, tipoPersona?: string) {
    this.nombre = nombre;
    this.apellido = apellido;
    this.cedula = cedula;
    this.fechaNacimiento = fechaNacimiento;
    this.tipoPersona = tipoPersona;
  }

}

File: ConcentraVHM.Domain/Entities/Cliente.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace ConcentraVHM.Domain.Entities
{
	public class Cliente
	{
        public string Nombre { get; set; }
        public string Apellido { get; set; }
        [Key]
        public string Cedula { get; set; }
        public DateTime FechaNacimiento { get; set; }
        public char TipoPersona { get; set; }


    }

    
}

